ozkour-front/src/stores/talks.js

```diff
@@ -62,6 +64,8 @@ export const useTalkStore = defineStore({
           },
           paramsSerializer: (params) => qs.stringify(params, { encode: false })
         })
+
+      this.retreiving_talks = false
```

Comment:
Dans ce genre de situation, il est souhaitable d'ajouter un try-catch et de mettre `this.retreiving_talks = false` dans le finally (que ca plante ou pas, il faut que la valeur passe à `false`)